src/containers/allelePage/AlleleMolecularConsequences.jsx

```diff
@@ -6,14 +6,13 @@ import LoadingSpinner from '../../components/loadingSpinner.jsx';
 import NoData from '../../components/noData.jsx';
 // import { VariantJBrowseLink } from '../../components/variant';
 import Subsection from '../../components/subsection.jsx';
-import VariantToTranscriptTable from './VariantToTranscriptTable.jsx';
+import VariantToTranscriptTableNew from './VariantToTranscriptTableNew.jsx';
```

Comment:
Can this be called curation vs "new" thats what we have done everywhere else

Reply:
I will rename this file back to VariantToTranscriptTable in the next step when I convert the summary section of the variant page.